rfcs/20240806-c-api/README.md

```diff
@@ -75,8 +75,8 @@ API, and the current oneCCL API.
 |`ncclResult_t ncclCommInitRank(comm, size, id, rank)`|`onecclResult_t onecclCommInitRank(comm, size, id, rank)(1)`|`comm cl::create_communicator(size, rank, device, context, kvs) comms ccl:create_communicators(size, rank, device, context, kvs)`|
 |`ncclResult_t ncclCommInitRankConfig(comm, size, id, rank, attr)`|`onecclResult_t onecclCommInitRankConfig(comm, size, id, rank, attr)`|`comm ccl:create_communicator(size, rank, device, context, kvs, attr)`|
 |`ncclResult_t ncclCommInitAll (comms, ndev, dev_list)`|`onecclResult_t onecclCommInitAll(comms,ndev,dev_list)`| Not currently available.Working on adding support.|
-|`ncclCommSplit`    |	Not implemented	              | Not implemented        |
-|`nccltResult ncclCommFinalize(comm)`|`onecclResult_t onecclCommFinalize(comm)`| N/A |
+|`ncclResult_t ncclCommSplit (comm, color, key, newcomm, config`    |	inecclCommSplit(comm, color, key, newcomm, config)	              | Not implemented        |
+|`ncclResult_t ncclCommFinalize(comm)`|`onecclResult_t onecclCommFinalize(comm)`| N/A |
```

Comment:
should be oneccl instead of ineccl

Reply:
commSplit is actually supported and implemented for the MPI transport. Maybe we shoudl not say Not Implemented